### DISCUSSION TITLE
Community Guidelines - Reach Amplification

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#15] [Rejected]
To find out if we would need `Community Guidelines - Reach Amplification` document type, let's list in this discussion the documents that would be defined by this type.

- Twitter: [About specific instances when a Tweet’s reach may be limited](https://help.twitter.com/en/rules-and-policies/twitter-reach-limited)

**Comment by MattiSG** [comment]
Thanks for this suggestion!

As explained in https://github.com/OpenTermsArchive/terms-types/discussions/13#discussioncomment-5629962, there are no named subtypes since https://github.com/OpenTermsArchive/engine/issues/773: the Community Guidelines are just one terms, made of several documents that are combined.

There will thus never be a “Community Guidelines - Reach Amplification” terms type. However, if you believe there should be a dedicated `Reach Amplification` type, please suggest is as a new type, with examples from two other services to demonstrate it is generic enough to be tracked :smiley:
